changelogs/unreleased/4398-ywk253100

```diff
@@ -0,0 +1 @@
+Fix #4009: the backup cannot be deleted after the application in uninstalled
```

Comment:
Shouldn't the changelog be without the issue ID?

Reply:
I'm not aware of this, will remove the issue ID. tks